superset/db_engine_specs/__init__.py

```diff
@@ -49,3 +49,8 @@
             and attribute.engine != ""
         ):
             engines[attribute.engine] = attribute
+
+            # populate engine alias name to engine dictionary
+            if isinstance(attribute.engine_aliases, tuple) and attribute.engine_aliases:
+                for engine_alias in attribute.engine_aliases:
+                    engines[engine_alias] = attribute
```

Comment:
This can probably be simplified as follows:
```suggestion
            # populate engine alias name to engine dictionary
            for engine_alias in attribute.engine_aliases or []:
                engines[engine_alias] = attribute
```